godotsteam.cpp

```diff
@@ -426,6 +426,11 @@ bool Steam::restartAppIfNecessary(uint32 app_id) {
 
 // Initialize the SDK, without worrying about the cause of failure.
 Dictionary Steam::steamInit(bool retrieve_stats, uint32_t app_id, bool embed_callbacks) {
+	// Use Project Setting as default (default for setting is 0)
+	if (app_id == 0) {
+		app_id = GLOBAL_GET("godot_steam/steam/app_id");
```

Comment:
In my local copy I changed `godot_steam/steam/` in all these to `steam/initialization/`.